infra/templates/README.md.jinja2

```diff
@@ -133,7 +133,7 @@ pprint(feature_vector)
 Please refer to the official documentation at [Documentation](https://docs.feast.dev/)
  * [Quickstart](https://docs.feast.dev/getting-started/quickstart)
  * [Tutorials](https://docs.feast.dev/tutorials/tutorials-overview)
- * [Running Feast with GCP/AWS](https://docs.feast.dev/how-to-guides/feast-gcp-aws)
+ * [Running Feast with Snowflake/GCP/AWS](https://docs.feast.dev/how-to-guides/feast-snowflake-gcp-aws)
```

Comment:
Is it just me, or are the changes to the readme and template different

Reply:
they are lol. Miles had skipped the template so I went back and fixed